app/controllers/recipes_controller.rb

```diff
@@ -0,0 +1,7 @@
+class RecipesController < ApplicationController
+    def show
+        @recipe = Recipe.find(params[:id])
+        @ingredients = @recipe.ingredients
```

Comment:
We don't need to create an `@ingredients` instance variable here. We can iterate over `@recipe.ingredients` in our view.